test/integration/master/transformation_testcase.go

```diff
@@ -90,10 +93,6 @@ func newTransformTest(l kubeapiservertesting.Logger, transformerConfigYAML strin
 		return nil, err
 	}
 
-	if e.secret, err = e.createSecret(testSecret, e.ns.Name); err != nil {
```

Comment:
was this intentionally dropped?

Reply:
Yes, it was.
I moved the responsibility of creating a secret under test to Tests.
That is to accommodate testing of scenarios where kube-apiserver/kms-plugin are intentionally configured to be broken - testing of healthz negative test.